test/e2e/catalogutil/catalog.go

```diff
@@ -1350,7 +1351,7 @@ func createCSVTemplate(catalogEntry *CatalogEntry) (*operatorsv1alpha1.ClusterSe
 			},
 			DisplayName: catalogEntry.PackageName,
 			InstallStrategy: operatorsv1alpha1.NamedInstallStrategy{
-				StrategyName: "install",
+				StrategyName: v1alpha1.InstallStrategyNameDeployment,
```

Comment:
fixed the strategy name, previously it was failing with 

```
time="2021-03-16T16:15:13Z" level=info msg=syncing id=Zmf36 ip=install-697xl namespace=operators phase=Installing
time="2021-03-16T16:15:13Z" level=info msg="added to bundle, Kind=ClusterServiceVersion" configmap=operators/6195f7a3380649ce200ebcd38d56f3d75e3308b2d3809c0170ea7f7967b7e30 key=testoperatora-v1.0.1-csv.yaml
time="2021-03-16T16:15:13Z" level=info msg="added to bundle, Kind=CustomResourceDefinition" configmap=operators/6195f7a3380649ce200ebcd38d56f3d75e3308b2d3809c0170ea7f7967b7e30 key=testoperatoras.sample.ibm.com-crd.yaml
time="2021-03-16T16:15:13Z" level=debug msg="failed to unpack bundles: failed to turn bundle into steps: unrecognized install strategy"
```

Reply:
makes sense